src/assets/translations/en/main.json

```diff
@@ -2571,5 +2571,11 @@
     "emissionsPoolHelper": "Emissions Pool (This Epoch)",
     "foxBurnAmount": "FOX Burn Amount (This Epoch)",
     "pendingDistribution": "Pending Distribution"
+  },
+  "markets": {
+    "recommended": "Recommended",
+    "watchlist": "My Watchlist",
+    "categories": "Categories",
+    "marketsBody": "Nunc vel efficitur ligula, vel sagittis est. Morbi et sapien a ligula venenatis consequat."
```

Comment:
😭 😆

Reply:
<img width="726" alt="image" src="https://github.com/user-attachments/assets/0897c8db-d362-40f5-a8bf-c85ae9ff548a">
just honouring the spec ser